src/main/catalog/catalog-entity-registry.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { catalogEntityRegistry } from "./catalog-entity-registry";
+
+const catalogEntityRegistryInjectable = getInjectable({
+  id: "catalog-entity-registry",
+  instantiate: () => catalogEntityRegistry,
```

Comment:
Could we actually do it the other way around so that this isn't `causesSideEffects`.

Namely, actually create a new instance here and then make `catalogEntityRegistry` a `legacyGlobal...`?

Reply:
Sure, we could but do we want to do it here? We have tried to avoid expanding scope of this PR as much as possible. Changing that will introduce more files that change.